version.sbt

```diff
@@ -1 +1 @@
-ThisBuild / version := "0.0.40-SNAPSHOT"
+ThisBuild / version := "0.0.41-SNAPSHOT"
```

Comment:
Manually updating this one higher as there already exists an 0.0.40-SNAPSHOT in sonatype